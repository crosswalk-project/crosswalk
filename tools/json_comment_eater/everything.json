// Copyright 2013 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// Test API.
{ "namespace": "test",
  "comments": "yo",  // Comments all have a // in them.
  "strings": "yes",  // Comment with "strings" and " character
  "escaped\"": "string\"isescaped",
  "more//": "\"more",
  "so\\many": "\\\\escapes\\\\\"whoa",
  "comment//inmiddle": "of string"
}
