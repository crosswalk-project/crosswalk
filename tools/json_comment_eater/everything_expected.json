




{ "namespace": "test",
  "comments": "yo",  
  "strings": "yes",  
  "escaped\"": "string\"isescaped",
  "more//": "\"more",
  "so\\many": "\\\\escapes\\\\\"whoa",
  "comment//inmiddle": "of string"
}
